packages/pluggableWidgets/switch-web/CHANGELOG.md

```diff
@@ -0,0 +1,25 @@
+# Changelog
+All notable changes to this widget will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+### Added
+- Add `Brand Secondary` `Style` design-property option.
+- Add `aria-readonly` for improved accessibility.
+
+### Changed
+- Use `div` instead of `small` for improved semantics.
+- Convert `Style` and `Device style` properties to design-properties.
+- Label property converted to system property and moved to common tab.
+- Editability property converted to system property and moved to common tab.
+
+### Removed
+- Removed `input` element.
+- Removed `info` and `inverse` `Style` property options.
+- Removed label width property.
+
+## 3.0.0 DD-MM-YYY
```

Comment:
Why did you add this here?

Reply:
Shouldn't this be on line 6?

Reply:
@IIsaku  I don't know which date Switch is going to be released, as currently it depends on Leo's testing.

Reply:
Wouldn't it make change to list new changes under [Unreleased] and only create a header with version and date during the release process?

Reply:
Which is basically what you're doing. Except I also don't understand the need for this header

Reply:
@KevinVlaanderen I agree, the usual+ideal flow would be: 1) build up changes under `[unreleased]`, then after the widget is released 2) make a PR moving the changes from `[unreleased]` to proper section. 

However, since I did this work, and I know that it's going to be version 3.0.0, I got ahead of the game ;)

Reply:
I just don't know when it will actually be released. This relies on Leo.